From 88347d42c6ae2ecb7df31c159cac47e17e83935c Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.xyz>
Date: Wed, 12 Jun 2024 15:27:37 +0800
Subject: [PATCH 126/133] loongarch(loongarch.cc): add missing / at
 start-of-line for a comment

This commit fixes a previous bad merge.

Signed-off-by: Mingcong Bai <baimingcong@uniontech.com>
---
 gcc/config/loongarch/loongarch.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/config/loongarch/loongarch.cc b/gcc/config/loongarch/loongarch.cc
index 215f7cf0baa..3eacf59bbbb 100644
--- a/src/gcc/config/loongarch/loongarch.cc
+++ b/src/gcc/config/loongarch/loongarch.cc
@@ -7952,7 +7952,7 @@ loongarch_expand_vec_perm_even_odd_1 (struct expand_vec_perm_d *d, unsigned odd)
   return true;
 }
 
-* Pattern match extract-even and extract-odd permutations.  */
+/* Pattern match extract-even and extract-odd permutations.  */
 
 static bool
 loongarch_expand_vec_perm_even_odd (struct expand_vec_perm_d *d)
-- 
2.43.4

